Core/include/Acts/Surfaces/detail/DiscSurface.ipp

```diff
@@ -44,25 +44,25 @@ inline void DiscSurface::initJacobianToGlobal(const GeometryContext& gctx,
   const auto rframe = referenceFrame(gctx, position, direction);
 
   // special polar coordinates for the Disc
-  double lrad = pars[eLOC_0];
-  double lphi = pars[eLOC_1];
+  double lrad = pars[eBoundLoc0];
+  double lphi = pars[eBoundLoc1];
   double lcos_phi = cos(lphi);
   double lsin_phi = sin(lphi);
   // the local error components - rotated from reference frame
-  jacobian.block<3, 1>(0, eLOC_0) =
+  jacobian.block<3, 1>(0, eBoundLoc0) =
       lcos_phi * rframe.block<3, 1>(0, 0) + lsin_phi * rframe.block<3, 1>(0, 1);
-  jacobian.block<3, 1>(0, eLOC_1) =
+  jacobian.block<3, 1>(0, eBoundLoc1) =
       lrad * (lcos_phi * rframe.block<3, 1>(0, 1) -
               lsin_phi * rframe.block<3, 1>(0, 0));
   // the time component
-  jacobian(3, eT) = 1;
+  jacobian(3, eBoundTime) = 1;
```

Comment:
It's not relevant to this PR, but I just wonder why we use explicit index for the free parameters indices while we use the enum for the bound parameters here and elsewhere?

Reply:
There is no good reason. This should be fixed. Can you open an issue so we can review the usage of bare indices after `v1.0.0`?

Reply:
Sure, will do it.